src/Microphysics1M.jl

```diff
@@ -191,18 +191,17 @@ function terminal_velocity(
     # from D=125um to D=625um using B2 and D=625um to inf using B4.
     if q > FT(0)
         # coefficients from Table B4 from Chen et. al. 2022
-        aiu, bi, ciu = CO.Chen2022_vel_coeffs_B4(vel, ρ)
-        #aiu, bi, ciu = CO.Chen2022_vel_coeffs_B2(vel, ρ)
+        aiu, bi, ciu = CO.Chen2022_vel_coeffs_B4(vel, ρₐ, ρᵢ)
+        #aiu, bi, ciu = CO.Chen2022_vel_coeffs_B2(vel, ρₐ, ρᵢ)
```

Comment:
Why is this function call commented out?

Reply:
Thats a bigger TODO that we may or may not chose to address. The Chen 2022 velocity parameterization switches coefficient values between small and large ice particles (table B2 vs B4). Technically in 1M terminal velocity we should integrate by parts and use both tables to get the bulk velocities for cloud condensate and snow. But instead we use B2 for cloud ice and B4 for snow and keep the simple analytical formulations

Reply:
Since the TODO isn't addressed in this PR does it make sense to open an issue or leave a code comment to track it?